tmp/test.go

```diff
@@ -0,0 +1 @@
+package tmp
```

Comment:
Is this intentional? Or was it accidentally committed?

Reply:
Nope, should not have committed this. Thanks for catching.